lib/tasks/application.rake

```diff
@@ -122,4 +124,46 @@ namespace :application do
     Rails.logger = ActiveSupport::Logger.new($stdout)
     Rails.logger.level = 1
   end
+
+  namespace :links_valid do
+    desc "Checks the validity of links in division summary"
+    task divisions: :environment do
+      include PathHelper
+
+      md = Redcarpet::Markdown.new(Redcarpet::Render::HTML)
+      broken_url_constant = 999
```

Comment:
Having to do something like this is a bit of "code smell" https://en.wikipedia.org/wiki/Code_smell. I guess more deeply you're using `url_hash_table` to do two separate things. It's recording whether something's cached and whether it's a valid url.

Reply:
Fixed in PR #1318